gammapy/time/tests/test_lightcurve.py

```diff
@@ -28,3 +35,61 @@ def test_lightcurve_fvar():
 def test_lightcurve_plot():
     lc = LightCurve.simulate_example()
     lc.plot()
+
+
+@requires_data('gammapy-extra')
+@pytest.fixture(scope='session')
+def spec_extraction():
+    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    obs_ids = [23523, 23526, 23559, 23592]
+    obs_list = data_store.obs_list(obs_ids)
+    
+    target_position = SkyCoord(ra=83.63308,
```

Comment:
You can just use the fact that `frame='icrs'` is the default (we do in many places in Gammapy) and put this on one line:
```python
target_position = SkyCoord(ra=83.63308, dec=22.01450, unit='deg')
```